### PR TITLE
fix: Remove usages of deprecated fields and bump to Flutter stable

### PR DESCRIPTION
**Pull request by spydon**
Currently Flutter is pinned to an old version, this PR makes sure that we are always using the latest stable version of Flutter in the pipeline.

### CODE REVIEW

File: lib/src/widgets/property_widgets/color_property.dart [hunk 1]
```diff
@@ -63,7 +63,7 @@ class ColorPropertyState extends State<ColorProperty> {
       label: widget.property.name,
       child: ElevatedButton(
         style: ElevatedButton.styleFrom(
-          primary: currentColor,
+          backgroundColor: currentColor,
```
Comment (erickzanardo) [inline lib/src/widgets/property_widgets/color_property.dart:66]: Maybe this should be on a different PR? Cause this will need a release

Comment (spydon) [reply lib/src/widgets/property_widgets/color_property.dart:66]: It's catch-22, if we don't want to force merge this one, because the analyze on stable will fail without this fix.

Comment (spydon) [reply lib/src/widgets/property_widgets/color_property.dart:66]: I can just rename this branch to `fix: Remove usages of deprecated fields and bump to Flutter stable` 😄

Comment (erickzanardo) [reply lib/src/widgets/property_widgets/color_property.dart:66]: Renaming the PR works!

**Review by erickzanardo — APPROVED** [review]
